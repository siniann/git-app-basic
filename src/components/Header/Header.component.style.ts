import * as CSS from 'csstype';

const headerStyle: CSS.Properties = {
  background: '#333',
  color: '#fff',
  textAlign: 'center',
  padding: '5px',

};
export default headerStyle;
