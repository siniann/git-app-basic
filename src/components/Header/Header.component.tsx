import React from 'react';
import headerStyle from './Header.component.style';

function Header() {
  return (
    <div style={headerStyle}>
      <h1>
          Git Office
      </h1>
    </div>
  );
}

export default Header;
