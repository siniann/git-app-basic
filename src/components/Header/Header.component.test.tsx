import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Header from './Header.component';

describe('Header', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should contain the text Git Office', () => {
    const wrapper = mount(<Header />);

    expect(wrapper.text()).toContain('Git Office');
  });
});
