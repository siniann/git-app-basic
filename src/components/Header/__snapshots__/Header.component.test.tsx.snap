// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  style={
    Object {
      "background": "#333",
      "color": "#fff",
      "padding": "5px",
      "textAlign": "center",
    }
  }
>
  <h1>
    Git Office
  </h1>
</div>
`;
